src/BF/VTX/vtx_defaults.lua

```diff
@@ -0,0 +1,11 @@
+frequencyTable = {
+    { 5865, 5845, 5825, 5805, 5785, 5765, 5745, 5725 }, -- Boscam A
+    { 5733, 5752, 5771, 5790, 5809, 5828, 5847, 5866 }, -- Boscam B
+    { 5705, 5685, 5665, 5645, 5885, 5905, 5925, 5945 }, -- Boscam E
+    { 5740, 5760, 5780, 5800, 5820, 5840, 5860, 5880 }, -- FatShark
+    { 5658, 5695, 5732, 5769, 5806, 5843, 5880, 5917 }, -- RaceBand
+}
+frequenciesPerBand = 8
+bandTable = { [0]="U", "A", "B", "E", "F", "R" }
+deviceTable = { [1]="6705", [3]="SA", [4]="Tramp", [255]="None" }
+pitModeTable = { [0]="OFF", "ON" }
```

Comment:
Not sure if these need to be part of the 'defaults'. But then, on the other hand, it makes sense to have them in one centralised file instead of having them included the file for each TX type - we currently do too much of this anyway.

Reply:
@mikeller I already have a plan to refactor the scripts into unified logic, and just load the "layout" for the screens from files based on the screen dimensions, rather than the radio type, as all else seems to be the same - which should make maintenance much easier. I also noticed that the rx.lua script is missing from the X7 folder - do you know if this is deliberate?

Reply:
Re missing rx.lua file, I suspect that this is just a consequence of the copy / paste way that multi TX support is done at the moment.